coralproject/talk
# [CORL-2655] Fix no-text mode for host article count embed

## What does this PR do?

Only show host article count text if the text class is present. If they set the `data-coral-notext="true"` you will only get a comment count number from the count embed instead of `12 Comments / 4 New` (or similar).

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags? 

No

## If any indexes were added, were they added to `INDEXES.md`?

No new indexes.

## How do I test this PR?

- Spin up Coral
- In `multi-site-test` on `src/views/index.html` on line 28 in the `span` with `class="coral-count"`, set the attribute `data-coral-notext="true"`
- Spin up multi-site-test
- Go to your multi-site-test page
- See that the counts are just showing a number
- Remove the `data-coral-notext="true"` attribute
- See that it now shows count texts similar to `12 Comments / 4 New`
 
## How do we deploy this PR?

No special considerations.
